config/base/scene/simple_desktop.json

```diff
@@ -241,6 +238,67 @@
           "texture": "../share/resources/textures/neptune.jpg"
         }
       }
+    },
+	"csp-simple-wms-bodies": {
```

Comment:
formatting looks strange? Tabs!? Meybe only here on github?

Reply:
It is indentet with a tab instead of spaces.